src/quatrex/electron/solver.py

```diff
@@ -85,7 +85,10 @@ def __init__(
 
         self.bare_system_matrix += self.overlap_sparray
         scale_stack(self.bare_system_matrix.data[:], self.local_energies)
-        self.bare_system_matrix -= self.hamiltonian_sparray
+        self.eta = quatrex_config.electron.eta
+        self.bare_system_matrix -= (
+            self.hamiltonian_sparray - 1j * self.eta * self.overlap_sparray
+        )
```

Comment:
As i understand it, the eta should only be added for the boundary conditions

Reply:
Ah, okey. Do you the reason for only including it in the contacts?

Reply:
But the eta is normally added such that the retarded function has the correct causality properties? I don't think it should be considered as a broadening caused by scattering

Reply:
Should we just keep both options? Seems to be the case in OMEN as well :man_shrugging:

Reply:
Let's keep both!

I'll add an additional eta for the contacts.